storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc

```diff
@@ -0,0 +1,38 @@
+[id="persistent-storage-csi-ebs"]
+= GCP PD CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-gcp-pd
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Google Cloud Platform (GCP) persistent disk (PD).
+
+:FeatureName: GCP PD CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
+
+To create CSI-provisioned persistent volumes (PVs) that mount to GCP PD storage assets, {product-title} installs the GCP PD CSI Driver Operator and the GCP PD CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+
+* The _GCP PD CSI Driver Operator_ provides a storage class by default that you can use to create PVCs. You also have the option to create the GCP PD storage class as described in xref:../../storage/persistent_storage/persistent-storage-gce.adoc#persistent-storage-gce[Persistent storage using GCE Persistent Disk].
+
+* The _GCP PD driver_ enables you to create and mount GCP PD PVs.
+
+include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+{product-title} defaults to using an in-tree, or non-CSI, driver to provision GCP PD storage. This in-tree driver will be removed in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree driver are planned for migration to the CSI driver at that time.
+====
+
+// GCP PD CSI driver storage class parameters
+// Include a module that describes all GCP PD CSI storage class parameters supported by OCP
+// https://issues.redhat.com/browse/STOR-488
```

Comment:
This is the authoritative list: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver#createvolume-parameters

In addition, some `StorageClass.parameters` are handled by the external-provisioner sidecar, thus they're understood by any CSI driver. In GCE PD case it's basically only parameter `csi.storage.k8s.io/fstype`, as documented on https://kubernetes-csi.github.io/docs/external-provisioner.html#storageclass-parameters (GCE does not need any of the secret parameters specified there).

Reply:
Thanks. I've created a separate module in this PR (`modules/persistent-storage-csi-gcp-pd-storage-class-ref.adoc`) to cover the storage class parameters. I'm not sure I've got it explained quite right, so PTAL.